### ISSUE TITLE
Colorbox elements are inserted on every page view via JS

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/education.wsu.edu#6]
This causes an accessibility issue with an empty button and likely some other invalid HTML - https://dashboard.wsu.edu/accessibility/#/domain/education.wsu.edu/code/WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name

This should be loaded only on pages where it's needed.
